src/js/constants.js

```diff
@@ -5,3 +5,12 @@ export const LOTTO_PURCHASE_UNIT = 1000;
 
 export const MESSAGE_FOR_EMPTY_VALUE = '반드시 값을 입력해주세요!';
 export const MESSAGE_FOR_INVALID_UNIT_VALUE = '로또 한 장의 단위는 1000원 입니다.';
+export const MESSAGE_FOR_INVALID_WINNING_NUMBERS = '중복된 번호를 입력할 수 없습니다.';
+
+export const PRIZE_FOR_PLACE = {
+  place1: 2000000000,
+  place2: 30000000,
+  place3: 1500000,
+  place4: 50000,
+  place5: 5000,
```

Comment:
```suggestion
  place1: 2_000_000_000,
  place2: 30_000_000,
  place3: 1_500_000,
  place4: 50_000,
  place5: 5_000,
```
이렇게 표현하면 숫자를 읽을 때 더 수월하지 않을까요!?

Reply:
[0b004b8](https://github.com/next-step/js-lotto/pull/184/commits/0b004b8dc86dd1809a4162b7b0404db2288a51a8)로 변경하였습니다!
감사합니다!